keps/sig-network/2004-topology-aware-routing/README.md

```diff
@@ -0,0 +1,553 @@
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Terminology](#terminology)
+  - [Configuration](#configuration)
+  - [Notes/Constraints/Caveats](#notesconstraintscaveats)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Assumptions](#assumptions)
+  - [Kube-Proxy](#kube-proxy)
+  - [Balanced](#balanced)
+    - [Scheme Logic:](#scheme-logic)
+  - [Require](#require)
+    - [Scheme Logic:](#scheme-logic-1)
+  - [Prefer](#prefer)
+    - [Terminology](#terminology-1)
+    - [Scheme Logic](#scheme-logic-2)
+    - [Evaluation Tool](#evaluation-tool)
+    - [Test Results](#test-results)
+  - [Recommended Algorithm: Local](#recommended-algorithm-local)
+    - [Optimizations](#optimizations)
+    - [Distribution of EndpointSlices](#distribution-of-endpointslices)
+    - [Rationale](#rationale)
+    - [Potential Issues](#potential-issues)
+  - [Other Algorithms Evaluated](#other-algorithms-evaluated)
+    - [Local-Shared](#local-shared)
+      - [Rationale](#rationale-1)
+      - [Potential Issues](#potential-issues-1)
+    - [Local-Weighted](#local-weighted)
+      - [Rationale](#rationale-2)
+      - [Potential Issues](#potential-issues-2)
+    - [Local-Opt](#local-opt)
+      - [Rationale](#rationale-3)
+      - [Potential Issues](#potential-issues-3)
+  - [Test Plan](#test-plan)
+  - [Observability](#observability)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Kubernetes clusters are increasingly deployed in multi-zone environments but
+unfortunately network routing has not caught up with that. This doc proposes an
+automatic topology aware routing mechanism with a new Service annotation. This
```

Comment:
I'm assuming that the "topology routing mechanism" will also use the well known labels `topology.kubernetes.io/zone` https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesiozone , is that correct?
if affirmative can we add a small comment mentioning it?

Reply:
This a good point, thanks for the feedback! Added the usage of this label in the `Goal` section.